.github/workflows/ci.yml

```diff
@@ -299,14 +300,22 @@ jobs:
 
       - name: Login to Docker
         uses: docker/login-action@v1
-        if: env.DOCKER_USR != ''
+        if: env.CONTRIB_DOCKER_USR != ''
         with:
-          username: ${{ secrets.DOCKER_USR }}
-          password: ${{ secrets.DOCKER_PSW }}
+          username: ${{ secrets.CONTRIB_DOCKER_USR }}
+          password: ${{ secrets.CONTRIB_DOCKER_PSW }}
+
+      - name: Login to Upbound
+        uses: docker/login-action@v1
+        if: env.XPKG_ACCESS_ID != ''
+        with:
+          registry: xpkg.upbound.io
+          username: ${{ secrets.XPKG_ACCESS_ID }}
+          password: ${{ secrets.XPKG_TOKEN }}
```

Comment:
@chlunde @Duologic I don't have admin access to this repo but have these been configured already? I don't see them on the Marketplace (in the `crossplane-contrib` org), so I think we'll have to create the robot token there and then update these.

Reply:
@jastang has created the robot ID/token and I have added it as a new repository secret that should be available to this github action ✅